src/libdredd/src/mutation_replace_expr.cc

```diff
@@ -311,6 +312,7 @@ void MutationReplaceExpr::Apply(
   assert(!result && "Rewrite failed.\n");
   (void)result;
```

Comment:
Please sneak in a "keep release-mode compilers happy" comment similar to other `Apply` methods.

Reply:
Done.